app/routes/quotes/components/AddQuote.tsx

```diff
@@ -1,95 +1,108 @@
+import { Field } from 'react-final-form';
 import { Helmet } from 'react-helmet-async';
-import { Field, reduxForm } from 'redux-form';
-import { Button, TextEditor, withSubmissionError } from 'app/components/Form';
+import { Button, TextInput } from 'app/components/Form';
+import LegoFinalForm from 'app/components/Form/LegoFinalForm';
+import { withSubmissionErrorFinalForm } from 'app/components/Form/utils';
 import RandomQuote from 'app/components/RandomQuote/RandomQuote';
+import { spySubmittable, spyValues } from 'app/utils/formSpyUtils';
 import { createValidator, required } from 'app/utils/validation';
 import { navigation } from '../utils';
 import styles from './Quotes.css';
 
 type Props = {
-  addQuotes: (arg0: Record<string, any>) => Promise<any>;
-  invalid: boolean;
-  pristine: boolean;
-  submitting: boolean;
-  handleSubmit: (arg0: (arg0: Record<string, any>) => Promise<any>) => void;
+  addQuotes: (quote: { text: string; source: string }) => Promise<unknown>;
```

Comment:
Could you put the actual return type here? Or void if you're unsure. It's better then unknown when you know you don't use it IMO.